lib/utils/matrix_sdk_extensions/filtered_timeline_extension.dart

```diff
@@ -27,19 +27,75 @@ extension IsStateExtension on Event {
           type != EventTypes.RoomMember ||
           room.joinRules != JoinRules.public ||
           content.tryGet<String>('membership') == 'ban' ||
-          stateKey != senderId);
+          stateKey != senderId) &&
+      !isSomeoneChangeDisplayName() &&
+      !isSomeoneChangeAvatar() &&
+      !isGroupNameChangeWhenCreate() &&
+      !isGroupAvatarChangeWhenCreate() &&
+      !isJoinedByYourself() &&
+      !isActivateEndToEndEncryption() &&
+      !isInviteWhenCreate();
 
   static const Set<String> importantStateEvents = {
     EventTypes.Encryption,
     EventTypes.RoomCreate,
     EventTypes.RoomMember,
     EventTypes.RoomTombstone,
     EventTypes.CallInvite,
+    EventTypes.RoomName,
```

Comment:
Here i enable RoomName and RoomAvatar events, because if not, when someone change group name or group avatar, we can't get notify